docs/contents/releases/1-20/1/index.md

```diff
@@ -0,0 +1,10 @@
+# EKS-D v1-20-eks-1 Release
+
+For additional information, see the [changelog](CHANGELOG-v1-20-eks-1.md) for this release and the [EKS-D v1-20-eks-1](https://distro.eks.amazonaws.com/kubernetes-1-20/kubernetes-1-20-eks-1.yaml)
+release manifest, which defines the versions used in EKS-D.
+
+```yaml
+---
+- TODO
```

Comment:
The original intent of the manifest in here was a lazy way to show what all is included in a release. All we may need is almost exactly what is in the component versions above. Having links to what needs to be pulled would be nice too though

Reply:
We ended up just removing this whole thing and making it clearer where people can download it